pages.it/common/cmatrix.md

```diff
@@ -0,0 +1,20 @@
+# cmatrix
+
+> Genera lettere che cadono, come in Matrix.
+> Vedi anche: <https://github.com/abishekvashok/cmatrix>.
+
+- Attivare lo scorrimento asincrono:
+
+`cmatrix -a`
+
+- Rendi le lettere rosse
```

Comment:
```suggestion
- Rendi le lettere rosse:
```

Reply:
Clicked commit suggestion, should update momentarily with the colon